src/test/java/apoc/bolt/BoltTest.java

```diff
@@ -380,7 +390,9 @@ public void testLoadBigPathVirtual() throws Exception {
                 assertEquals(true, end.hasLabel(Label.label("Person")));
                 assertEquals("Anne", end.getProperty("name"));
                 assertEquals("Olsson", end.getProperty("surname"));
+                assertEquals(new InternalPoint3D(9157, 2.3, 4.5, 1.2), end.getProperty("born"));
```

Comment:
No importare staticamente:
`org.neo4j.values.storable.Values.pointValue`
e 
`org.neo4j.values.storable.CoordinateReferenceSystem`

dopo di che se è un cartesiano:
```suggestion
                assertEquals(pointValue(Cartesian, 1.0, 2.0), end.getProperty("born"));
```
```suggestion
                assertEquals(pointValue(Cartesian_3D, 1.0, 2.0, 3.0), end.getProperty("born"));
```
se è geospaziale:
```suggestion
                assertEquals(pointValue(WGS84, 1.0, 2.0), end.getProperty("born"));
```
```suggestion
                assertEquals(pointValue(WGS84_3D, 1.0, 2.0, 3.0), end.getProperty("born"));
```